pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -222,6 +237,32 @@ func (f *framework) RunPrefilterPlugins(
 	return nil
 }
 
+// RunFilterPlugins runs the set of configured filter plugins for pod on each candidate host.
+// If any of these plugins returns error, the node is not suitable for running pod.
+// Meanwhile, the failure message and status are set for each non-candidate host.
+func (f *framework) RunFilterPlugins(pc *PluginContext,
+	pod *v1.Pod, nodeName string) *Status {
+
+	_, exist := f.nodeInfoSnapshot.NodeInfoMap[nodeName]
+	if !exist {
+		msg := fmt.Sprintf("RunFilterPlugins: Fail to find NodeInfo in snapshot for name %s", nodeName)
+		klog.Error(msg)
```

Comment:
nit: It's better to use `Errorf` for this case since the msg is not used elsewhere.

Reply:
Done.